interface-definitions/system-login.xml.in

```diff
@@ -36,6 +36,68 @@
                     </properties>
                     <defaultValue>!</defaultValue>
                   </leafNode>
+                  <node name="otp">
+                    <properties>
+                      <help>2FA OTP authentication parameters</help>
+                    </properties>
+                    <children>
+                      <leafNode name="key">
+                        <properties>
+                          <help>Token Key Secret key for the token algorithm (see RFC 4226)</help>
+                          <valueHelp>
+                            <format>txt</format>
+                            <description>OTP key (base32 encoded secret)</description>
+                          </valueHelp>
+                          <constraint>
+                            <regex>[A-Z2-7]{20,10000}</regex>
```

Comment:
Lowercase is not supported? I tend to prefer lowercase over uppercase.

Reply:
BASE32 alphabet contains only capital letters and numbers 2-7:
https://en.wikipedia.org/wiki/Base32#RFC_4648_Base32_alphabet

Reply:
Not a fan of upper case letters in the CLI - can you please enable then both upper and lowercase? DUring rendering all can be converted to uppercase to match with BASE32

Reply:
Done